go.mod

```diff
@@ -30,6 +33,140 @@ require (
 	sigs.k8s.io/yaml v1.2.0
 )
 
+require (
```

Comment:
[nit] Should this be part of the same `require` block as above?

Reply:
This was autogenerated by `go mod tidy`. I don't write enough Golang to know what would be idiomatic.